src/authorization/AuthorizationsDefinition.ts

```diff
@@ -3543,6 +3543,110 @@ export const AUTHORIZATION_DEFINITION: AuthorizationDefinition = {
       },
     ]
   },
+  external: {
+    grants: [
+      {
+        resource: Entity.PAYMENT_INTENT, action: [Action.SETUP, Action.RETRIEVE]
+      },
+      {
+        resource: Entity.TAG, action: Action.LIST,
+        condition: {
+          Fn: 'custom:dynamicAuthorizations',
+          args: {
+            asserts: [],
+            filters: ['OwnUser'],
+            metadata: {
+              id: {
+                visible: false
+              }
+            },
+          }
+        },
+        attributes: [
+          'userID', 'active', 'description', 'visualID', 'issuer', 'default',
+          'createdOn', 'lastChangedOn'
+        ],
+      },
+      {
+        resource: Entity.TAG, action: Action.READ,
+        condition: {
+          Fn: 'custom:dynamicAuthorizations',
+          args: {
+            asserts: [],
+            filters: ['OwnUser'],
+          }
+        },
+        attributes: [
+          'userID', 'issuer', 'active', 'description', 'visualID', 'default',
+          'user.id', 'user.name', 'user.firstName', 'user.email', 'user.issuer'
+        ],
+      },
+      { resource: Entity.TRANSACTION, action: [Action.GET_ACTIVE_TRANSACTION, Action.READ],
+        condition: {
+          Fn: 'custom:dynamicAuthorizations',
+          args: {
+            asserts: [],
+            filters: ['OwnUser']
+          }
+        },
+        attributes: [
+          'id', 'chargeBoxID', 'timestamp', 'issuer', 'stateOfCharge', 'timezone', 'connectorId', 'status', 'meterStart', 'siteAreaID', 'siteID', 'companyID',
+          'currentTotalDurationSecs', 'currentTotalInactivitySecs', 'currentInstantWatts', 'currentTotalConsumptionWh', 'currentStateOfCharge',
+          'currentCumulatedPrice', 'currentInactivityStatus', 'roundedPrice', 'price', 'priceUnit','currentCumulatedRoundedPrice', 'stop.timestamp', 'stop.roundedPrice', 'stop.priceUnit', 'stop.totalDurationSecs', 'stop.totalConsumptionWh'
+        ]
+      },
+      { resource: Entity.SETTING, action: Action.READ },
+      { resource: Entity.TRANSACTION, action: [Action.REMOTE_STOP_TRANSACTION],
+        condition: {
+          Fn: 'custom:dynamicAuthorizations',
+          args: {
+            asserts: [],
+            filters: ['OwnUser']
+          }
+        }
+      },
+      { resource: Entity.CHARGING_STATION,
+        action: [Action.REMOTE_START_TRANSACTION, Action.REMOTE_STOP_TRANSACTION, Action.START_TRANSACTION, Action.STOP_TRANSACTION, Action.AUTHORIZE, Action.READ],
```

Comment:
Entity.CHARGING_STATION, action: [Action.READ]
Should be separate from other actions and should contain attributes, currently the external use can read ALL fields of a CS.

Reply:
thanks ! done ✅